api/routers/metrics/area_metrics.py

```diff
@@ -58,10 +24,7 @@ def get_area_occupancy_hourly_report(areas: str = "", date: date = Query(date.to
     Returns a hourly report (for the date specified) with information about the occupancy in
     the areas <areas>.
     """
-    areas = get_areas(areas)
-    for area in areas:
-        validate_area_existence(area)
-    return OccupancyMetric.get_hourly_report(areas, date)
+    return get_hourly_metric("areas", areas, "occupancy", date)
```

Comment:
For all these constant strings I would prefer if we have them as constants somewhere and use those constants instead or repeating the string everywhere. I think we have a constants file, right? But it could go elsewhere

Reply:
Agree. I didn't include them in the constant file because it's not used in the rest of the project (device names, model names, etc are always strings). However, put them in the file make more sense.

I will change it.

Reply:
Done